docs/installation/installing-tentacles/polling-tentacles-web-sockets.md

```diff
@@ -0,0 +1,91 @@
+```powershell
+.\Tentacle.exe register-with --instance MyInstance --server "https://MyServer/"  --server-web-socket "wss://MyServer:443/OctopusComms" --comms-style TentacleActive --apikey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --environment "Test" --role "Web" 
+```
+
+### Changing an Existing Tentacle
+To change an existing tentacle to poll using WebSockets, run the following commands:
+```powershell
+.\Tentacle.exe service --instance MyInstance --stop
+.\Tentacle.exe configure --reset-trust
+.\Tentacle.exe register-with --instance MyInstance --server "https://MyServer/" --server-web-socket "wss://MyServer:443/OctopusComms" --comms-style TentacleActive --apikey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --environment "Test" --role "Web" 
+.\Tentacle.exe service --instance MyInstance --start
+```
+
+### High Availablity
+When issuing the `poll-server` command to add additional nodes to poll, `--server-comms-port` parameter and specify the `--server-web-socket <address>` parameter. For example:
+
+```powershell
+poll-server --instance MyInstance --server "https://MyServer/"  --server-web-socket "wss://MyServer:443/OctopusComms" --apikey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO"
+```
+## More Information
+
+### SSL Certificate
+Windows will need to be configured with an SSL certificate on the select address and port. Usually this is done by the other application sharing the port.
+The certificate does _not_ need to be trusted by the server or tentacle machine. The certificate also does not need to match the hostname.
+Self signed certificates are accepted, no chain of trust is required. It does need to be installed into the Personal certificate store of the Machine account.
+
+If you need to generate a self signed certificate, this can be done by issuing the following PowerShell command. Take note of the thumbprint generated.
+```powershell
+New-SelfSignedCertificate -Subject "CN=Example Website" -CertStoreLocation "Cert:\localMachine\My" -KeyExportPolicy Exportable
+```
+
+If your chosen certificate has not yet been associated with the selected address and port, use the `netsh` tool to install it. For example:
+```powershell
+netsh http add sslcert ipport=0.0.0.0:443 certhash=966857B08601B9ACA9A9F10E7D469AC521E2CD4B appid='{00112233-4455-6677-8899-AABBCCDDEEFF}'
+```
+```console
+netsh http add sslcert ipport=0.0.0.0:443 certhash=966857B08601B9ACA9A9F10E7D469AC521E2CD4B appid={00112233-4455-6677-8899-AABBCCDDEEFF}
+```
+For more details instructions, see Microsoft's [certificate HowTo](https://msdn.microsoft.com/en-us/library/ms733791(v=vs.110).aspx).
+
+### Thumbprint
+Unlike other tentacle configurations, the tentacle must be configured to trust the thumbprint of the SSL certificate and not the Octopus server itself. This is due to HTTPSYS performing the certificate exchange (not the Octopus Server) and then delegating the connection. Both the tentacle and server still verify the certificate thumbprint match the trusted thumbprint.
```

Comment:
This isn't as clear as it could be. Maybe it needs to distinguish between the common case (the portal is on 443 and we want to setup websockets), and the less common case (we are sharing with some other random application).